components/js/common.js

```diff
@@ -0,0 +1,7 @@
+import { GoogleAnalytics } from './google/analytics';
+
+export function CommonJS() {
+    // please change the following with the ab506.church trackging Id
```

Comment:
Add `// TODO: ` to the beginning of the comment so that it is easier to search for it using `grep` or `ag`

Reply:
done